tests/test_health_plugin.py

```
@@ -338,6 +340,31 @@ async def run_test():
+        asyncio.run(run_test())
+
+    def test_handle_room_message_exception_handler(self):
+        """Test exception handler in handle_room_message (lines 171-174)."""
+        self.plugin.matches = MagicMock(return_value=True)
+        self.plugin.generate_response = MagicMock(side_effect=RuntimeError("boom"))
+
+        room = MagicMock()
+        room.room_id = "!test:matrix.org"
+        event = MagicMock()
+
+        async def run_test():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)